### PR TITLE
Fix typo in -impredicative-set parsing

### PR DESCRIPTION
**Pull request by artagnon**

### CODE REVIEW

File: controller/coq_lsp.ml [hunk 1]
```diff
@@ -318,7 +318,6 @@ let dispatch_request ~method_ ~params : RAction.t =
   (* Lifecyle *)
   | "initialize" ->
     LIO.trace "dispatch_request" "duplicate initialize request! Rejecting";
-    (* XXX what's the error code here *)
```
Comment (ejgallego) [inline controller/coq_lsp.ml:321]: Actually that's a real TODO, `-32600` is likely not the right code.

Why do we mix unrelated changes in PRs?

**Review by ejgallego — CHANGES_REQUESTED** [review]
Thanks a lot for the fix, however the unrelated change needs to be removed (or an issue opened) as it is a real TODO.

**Comment by ejgallego** [comment]
I guess a changelog would be also useful, users may wonder why it was not working before.

**Comment by ejgallego** [comment]
Also we could add an entry for this in `examples` (or tests very soon I hope!)

**Comment by artagnon** [comment]
We can't add an example, as it requires a `_CoqProject` to be at the root of the repository.

**Review by ejgallego — APPROVED** [review]

**Comment by ejgallego** [comment]
> We can't add an example, as it requires a `_CoqProject` to be at the root of the repository.

I guess we can add subdirs in `example`, let's discuss about this in person as it is really related to #15

**Pull request merged**